src/components/ClusterManager.js

```diff
@@ -293,7 +293,7 @@ export default class ClusterManager extends PureComponent {
       (activeClusters.length > 1 || activeDisks.length > 1) && h(Link, {
         style: { marginRight: '1rem' },
         href: Nav.getLink('clusters'),
-        tooltip: 'Multiple runtimes and/or persistent disks found in this billing project. Click to select which to delete.'
+        tooltip: 'Multiple cloud environments found in this billing project. Click to select which to delete.'
```

Comment:
This just says cloud environments instead of runtimes and/or persistent disks now.